csharp/ql/test/library-tests/csharp10/FileScopedNamespace.cs

```diff
@@ -0,0 +1,8 @@
+namespace MyFileScopedNamespace;
```

Comment:
I would prefer a newline after this line.

Reply:
Yes, of course - VS code "cheated" me with the "reference" link (then it looks like there is an empty line unless you directly at the line numbers)